requirements.txt

```diff
@@ -1,2 +1,3 @@
 qiskit-terra>=0.16
+qiskit>=0.25.0
```

Comment:
This isn't actually correct, you don't depend on the qiskit metapackage (like you don't need aer to use the ionq provider). If there is a compatibility issue here (although I'm not sure what the issue is) it would be with terra. You should just bump the line above to be `qiskit-terra>=0.17.0`

Reply:
My mistake! Looks like I had some confusion internally, this happens in results processing when unexpected keyword args are passed through. I'll update this to pin terra.